### ISSUE TITLE
Date sorting doesn't work quite right

### ISSUE THREAD

**Issue by EmberHeartshine** [EmberHeartshine/RedHeart#4]
Title, date sorting only sorts year because of the way `parseInt()` works. Look into maybe changing date format to epoch then sort?

Code below related
``` javascript
var dateInt = 1600700978126;
var dateStr = "2020-09-21T15:09:38.126Z";

if ( isNaN( new Date( dateStr ) ) ) {
    console.log("String Error");
} else if ( isNaN( new Date( dateInt ) ) ) {
    console.log("Int Error");
} else {
    console.log( "No Error" );
}

Date.parse( dateStr );
new Date( dateInt ).valueOf();
//Both of the above will return 1600700978126 as an int

new Date( dateStr ).valueOf();
//The above will also return 1600700978126, but less efficiently than Date.parse()
```

**Comment by EmberHeartshine** [comment]
Some testing done; I need to figure out a way to differentiate between `parseInt( "2020-01-01" )` (which returns `2020` as an int) and `13` as an int. The easy way out is to check if the score is > 2000, but there might be competitions where the scores get that high. Perhaps force all date strings in the json to be in epoch time and parse it when reading to the table?

**Comment by EmberHeartshine** [comment]
Fixed in 4087eff
